### PR TITLE
Added System Dashboard

### LINKED ISSUE #15: Add system dashboards for MetricsHub

### PR DESCRIPTION
**Pull request by MohammedSentry**

### CODE REVIEW

File: src/dashboards/System/System (MetricsHub).json [hunk 1]
```diff
@@ -0,0 +1,2849 @@
...
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_METRICSHUB}"
+          },
+          "editorMode": "code",
+          "expr": "sum (system_memory_utilization_ratio{host_id=\"$host_id\", system_memory_usage=\"used\", site=\"$site\"})",
```
Comment (NassimBtk) [inline src/dashboards/System/System (MetricsHub).json]: `"expr": "sum (system_memory_utilization_ratio{host_id=\"$host_id\", system_memory_state=\"used\", site=\"$site\"})",`

File: src/dashboards/System/System (MetricsHub).json [hunk 2]
```diff
@@ -0,0 +1,2849 @@
...
+      "title": "Memory Usage",
+      "transformations": [
+        {
+          "id": "organize",
+          "options": {
+            "excludeByName": {},
+            "includeByName": {},
+            "indexByName": {},
+            "renameByName": {
+              "{__name__=\"system_memory_utilization_ratio\", agent_host_name=\"mk-win.internal.sentrysoftware.net\", connector_id=\"Linux\", host_id=\"mk-deb2\", host_name=\"mk-deb2.internal.sentrysoftware.net\", host_type=\"compute\", id=\"memory_usage\", os_type=\"linux\", site=\"Paris\", system_memory_limit=\"2063106048\", system_memory_usage=\"used\"}": "Used memory"
```
Comment (NassimBtk) [inline src/dashboards/System/System (MetricsHub).json:1678]: `"{__name__=\"system_memory_utilization_ratio\", agent_host_name=\"mk-win.internal.sentrysoftware.net\", connector_id=\"Linux\", host_id=\"mk-deb2\", host_name=\"mk-deb2.internal.sentrysoftware.net\", host_type=\"compute\", id=\"memory_usage\", os_type=\"linux\", site=\"Paris\", system_memory_limit=\"2063106048\", system_memory_state=\"used\"}": "Used memory"`

Comment (NassimBtk) [reply src/dashboards/System/System (MetricsHub).json:1678]: We need to rename the attribute key from `system_memory_usage` to `system_memory_state`

File: src/dashboards/System/System (MetricsHub).json [hunk 3]
```diff
@@ -0,0 +1,2849 @@
...
+      },
+      "pluginVersion": "10.4.2",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_METRICSHUB}"
+          },
+          "editorMode": "code",
+          "expr": "system_memory_usage_bytes{host_id=~\"$host_id\", site=\"$site\", system_memory_usage=~\"cached|buffers\"}",
```
Comment (NassimBtk) [inline src/dashboards/System/System (MetricsHub).json]: `"expr": "system_memory_usage_bytes{host_id=~\"$host_id\", site=\"$site\", system_memory_state=~\"cached|buffers\"}",`

Comment (NassimBtk) [reply src/dashboards/System/System (MetricsHub).json]: We need to rename the attribute key from `system_memory_usage` to `system_memory_state`

File: src/dashboards/System/System (MetricsHub).json [hunk 4]
```diff
@@ -0,0 +1,2849 @@
...
+          "instant": false,
+          "legendFormat": "{{system_memory_usage}}",
```
Comment (NassimBtk) [inline src/dashboards/System/System (MetricsHub).json:1779]: Should be `"legendFormat": "{{system_memory_state}}",`

File: src/dashboards/System/System (MetricsHub).json [hunk 5]
```diff
@@ -0,0 +1,2849 @@
...
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_METRICSHUB}"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "system_memory_usage_bytes{host_id=~\"$host_id\", site=\"$site\" }",
+          "instant": true,
+          "legendFormat": "{{system_memory_usage}}",
```
Comment (NassimBtk) [inline src/dashboards/System/System (MetricsHub).json]: Should be `"legendFormat": "{{system_memory_state}}",`

File: src/dashboards/System/System (MetricsHub).json (hunk 1 above)
Comment (NassimBtk) [reply src/dashboards/System/System (MetricsHub).json]: We need to rename the attribute key from `system_memory_usage` to `system_memory_state`.

**Review by bertysentry — CHANGES_REQUESTED** [review]
Please update the dashboard, following the fix of our system connectors 😬🤷‍♂️

**Review by bertysentry — APPROVED** [review]

**Pull request merged**